packages/lab-extension/src/index.ts

```diff
@@ -124,9 +127,45 @@ const openRetro: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/**
+ * A plugin to add a command to open the RetroLab Tree.
+ */
+const launchRetroTree: JupyterFrontEndPlugin<void> = {
+  id: '@retrolab/lab-extension:launch-retrotree',
+  autoStart: true,
+  requires: [ITranslator],
+  optional: [IMainMenu, ICommandPalette],
+  activate: (
+    app: JupyterFrontEnd,
+    translator: ITranslator,
+    menu: IMainMenu | null,
+    palette: ICommandPalette | null
+  ): void => {
+    const { commands } = app;
+    const trans = translator.load('jupyterlab');
+    const category = trans.__('Help');
+
+    commands.addCommand(CommandIDs.launchRetroTree, {
+      label: trans.__('Launch RetroLab Tree'),
```

Comment:
I think that `Tree` is not a noun that everyone associates with notebooks and might be confusing for some users. I se that it makes sense for an advanced user, but beginners and intermediate users might ignore this option thinking that it is not what they are looking for. What about naming it "RetroLab Notebook" for similarity with the existing "Launch Classic Notebook" entry?

```suggestion
      label: trans.__('Launch RetroLab Notebook'),
```

Reply:
I think you may be right about "Tree" but the problem is there are two routes to Retro now - (1) from the Help menu which jumps to the tree, and also (2) a link from any open notebook in the JupyterLab window.

And in that context, 'Launch RetroLab Notebook' sounds like you're going to jump to a notebook (2) rather than the root tree (1).

How about just "Launch RetroLab" (which is perhaps ambiguous, but at least not suggesting the opposite of which link above it really means)?

Or "Launch RetroLab File Browser"...

Reply:
Agree, both of the counterproposals sound good

Reply:
Great, renamed to File Browser